docs/02-dsp-ontologies/knora-base.md

```diff
@@ -369,11 +368,11 @@ a `kb:FileValue`. The base class `FileValue`, which is not intended to be used d
 - `originalMimeType` (0-1): The original MIME type of the file when it was uploaded to the Knora API server.
 
 - `isPreview` (0-1): A boolean indicating whether the file is a preview, i.e. a small image representing the contents of
-  the file. A preview is always a `StillImageFileValue`, regardless of the type of the enclosing `Representation`.
+  the file. A preview is always a `StillImageAbstractFileValue`, regardless of the type of the enclosing `Representation`.
 
 The subclasses of `FileValue`, which are intended to be used directly in data, include:
 
-- `StillImageFileValue`: Contains metadata about a still image file.
+- `StillImageAbstractFileValue`: Contains metadata about a still image file, which can be either `StillImageFileValue` (an image stored in Sipi) or `StillImageExternalFileValue` (a reference to an image stored in an external IIIF service).
```

Comment:
I'm ok with it like this, but I would argue that the two subclasses would also warrant their own bullet point